spec/lib/work_factory_spec.rb

```diff
@@ -19,14 +19,14 @@
       ],
       "uri": [
```

Comment:
You have a new strings being converted to symbol here that dont need to be wrapped in speech marks. Could you remove the quotes and just use symbols :)